joplin/css/admin.scss

```diff
@@ -38,10 +38,29 @@
 
 .page-explorer header,
 .homepage header,
-.page-editor header {
+.page-editor header:not(.hasform) {
```

Comment:
I'm curious as to how the hasform logic works, it's not clear to me when/where/why it's being used.

Reply:
so every page is a form, and modals can have forms too. For whatever reason in vanilla wagtail this attribute (.hasform) only pops up on modals, so its a good way to keep hiding headers elsewhere but un-hide them for modals. The logic of why or what else its used for I'm not sure about.

Reply:
is this the most specific selector we have available? if so, a comment along the lines of:
"we need to select on page chooser modals only, and this is the most specific selector I could find" would be more descriptive.